news/2021-12-27-clickers-choice-2021.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: "Clicker's Choice: Top 20 osu! Players of 2021"
+date: 2021-12-27 09:00:00 +0000
+---
+
+Welcome aboard a brand new voting platform by helping decide who the top 20 best osu! players of 2021 are!
```

Comment:
```suggestion
Welcome aboard a brand new platform to help you vote for the top 20 osu! players of 2021!
```

I would suggest changing the catchphrase, it's kind of weirdly structured @Ephemeralis